packages/embeds/embed-core/src/embed.ts

```diff
@@ -6,8 +6,9 @@ import type { InterfaceWithParent, interfaceWithParent, PrefillAndIframeAttrsCon
 import css from "./embed.css";
 import { SdkActionManager } from "./sdk-action-manager";
 import type { EventData, EventDataMap } from "./sdk-action-manager";
-import allCss from "./tailwind.generated.css?inline";
+import tailwindCss from "./tailwindCss";
```

Comment:
Added a proxy in b/w to import tailwindCss. This is because vitest doesn't allow importing non-existent modules(or so it seems). 

Now, I could mock tailwindCss.ts